app/locale/en_US/Mage_Newsletter.csv

```diff
@@ -88,6 +89,7 @@
 "Save Newsletter","Save Newsletter"
 "Save Template","Save Template"
 "Save and Resume","Save and Resume"
+"Security","Security"
```

Comment:
To add:
"<strong style="color:red">Important!</strong> Enabling this option means that your custom templates for newsletter subscription must contain the <em>form_key</em> post parameter. Otherwise newsletter subscription will not work."

Not sure how linefeeds are handled in `CDATA` and translation.

Reply:
I'm gonna add the translation and remove the line breaks altogether.